components/styled/icon/src/molecules/emotion-icon.tsx

```diff
@@ -0,0 +1,25 @@
+import { EmotionIconType, IconStatus, IconDirection } from '@ltht-react/styles'
+import { FC, HTMLAttributes } from 'react'
+import ChevronDoubleIcon from '../atoms/emotion-icons/chevron-double-icon'
+
+const EmotionIcon: FC<EmotionIconProps> = ({ type, size, status, direction, counterValue, ...rest }) => {
+  switch (type) {
+    case 'chevron-double': {
+      return <ChevronDoubleIcon size={size} direction={direction ?? 'up'} {...rest} />
+    }
+
+    default: {
+      return <>?</>
+    }
+  }
+}
+
+export interface EmotionIconProps extends HTMLAttributes<SVGElement> {
+  type: EmotionIconType
+  size: string | number
+  status?: IconStatus
+  direction?: IconDirection
+  counterValue?: number
+}
+
+export default EmotionIcon
```

Comment:
I'm not sure this molecule is really necessary - we're already abstracting fontawesome in our `*-icon` atoms, so whether the icon comes from fontawesome or emotion doesn't really matter as it's abstracted in both cases.

Reply:
emotion icon take size as string or number where as the font awesome have different size type.

Reply:
Yeah but they run our custom size type through a `calculateIconSize` method to translate into `lg` `sm`. We could create a similar method so the consumer can use them both the same way